shapeshift/web
# feat: make zrx swapper wallet agnostic

## Description

Upgrades zrx swapper to the wallet-agnostic pattern as part of our move to an SDK style integration of swappers in web.

## Pull Request Type


## Issue (if applicable)

progresses https://github.com/shapeshift/web/issues/5024 (but doesn't close it)

## Risk

High risk of broken swappers, especially zrx. Specifically trade execution.

## Testing

* Check all swappers still work
* Check zrx trades still work

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)
![image](https://github.com/shapeshift/web/assets/125113430/9f09d9c6-8c65-482d-b8c6-3158144ea9fb)
